src/popup.css

```diff
@@ -4,13 +4,13 @@ img {
 .img-container{
```

Comment:
Adding in bootstrap changes the default `h1` and `h2` font quite a lot. In my option, the bootstrap font is a little too big and not bold enough. Visiting sites like www.bestbuy.com causes the popup to stretch out because the heading takes up too much space. Also the `hr` is a lot lighter.

Can you add in the following code so it forces the fonts to resemble the older vanilla style? Or feel free to style it how you like, as long as it resembles the older format.

```
h1 {
    font-family: "Segoe UI", Tahoma, sans-serif;
    font-size: 2em;
    font-weight: bold;
}
h2 {
    font-family: "Segoe UI", Tahoma, sans-serif;
    font-size: 1.5em;
}
hr {
    border-top: 1px solid black;
}
```